botorch/models/gpytorch.py

```diff
@@ -48,6 +48,7 @@
 from gpytorch.likelihoods.gaussian_likelihood import FixedNoiseGaussianLikelihood
 from linear_operator.operators import BlockDiagLinearOperator, CatLinearOperator
 from torch import Tensor
+from torch._C import _get_tracing_state
```

Comment:
I am not a fan of importing a private method from a private module. Is there a way to reproduce this using public methods?

Reply:
This issue seems relevant: https://github.com/pytorch/pytorch/issues/27265

Reply:
Thanks for the guidance. This has been fixed now.